app-backend/build.sbt

```diff
@@ -117,3 +118,16 @@ lazy val database = Project("database", file("database"))
   .settings({
    libraryDependencies ++= slickDependencies ++ dbDependencies ++ loggingDependencies ++ Seq(Dependencies.akkaHttpExtensions)
   })
+
+lazy val tiler = Project("tileserver", file("tileserver"))
+  .settings(commonSettings:_*)
```

Comment:
That sounds good, I'm not really up on `sbt` but sounds like that improves organization.